geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderQueue.java

```diff
@@ -530,7 +549,9 @@ public void addShadowPartitionedRegionForUserPR(PartitionedRegion userPR) {
           addOverflowStatisticsToMBean(cache, prQ);
 
           // Wait for buckets to be recovered.
-          prQ.shadowPRWaitForBucketRecovery();
+          if (this.asyncEvent || sender.isPersistenceEnabled()) {
```

Comment:
Why are we only waiting for async or persistence when before it was in all cases?

Reply:
Previously, this was executed only at first start of GW sender (since at stop region was not closed).

Reply:
This part is delated, since we are not closing region at stop any more.